packages/card/index.test.js

```diff
@@ -0,0 +1,28 @@
+import { render } from '@testing-library/react'
+import Card from '.'
+
+describe('<Card />', () => {
+  it('should render the provided React children', () => {
```

Comment:
Some good basic test would be testing the card with props version renders each piece of content, so heading is in the document, description is in the document, the link is correct, etc.

Reply:
Yep, good call.  Additionally, perhaps some tests checking for the presence of composable children's content if added?  Tests for that can check to see if any props were passed to the `<Card />`; given how we've set it up, that test should fail if there are props being passed _and_ composable children being used.  

The TypeScript typing has been set up to cover this though, so this might be a case of testing the implementation details (which we probably want to avoid?)

Reply:
@alexcarpenter Wrote a suite of prop render tests here:
https://github.com/hashicorp/react-components/pull/694/commits/14a448c879f47fbc19aa21486cca5bf5ba7fea53

In writing these tests I was primarily pulling from conventions found elsewhere in our codebase; any feedback on methods used here is appreciated 🙏

Reply:
Things are looking good! Recorded a bit of a brain dump: https://hashicorp.slack.com/archives/C031LA4KS4S/p1660856664941599

Might be helpful to have a test ensuring the link is in the card too.

Reply:
Just pushed tests covering the link, and also setting heading levels in `<Card.Heading />`. I've replied in Slack on that brain dump, some great insights there!